src/components/main-content.tsx

```diff
@@ -0,0 +1,37 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./main-content.sass";
+import { InvestigateDialogComponent } from "./investigate-dialog";
+import { InvestigationPanelComponent } from "./investigation-panel";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class MainContentComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="main-content">
+        {this.renderDialog()}
+      </div>
+    );
+  }
+
+  private renderDialog() {
+    const {ui} = this.stores;
+    const {showInvestigationModalSelect} = this.stores.ui;
+    const {showInvestigationPanel} = this.stores.ui;
+    // {return ui.showInvestigationModalSelect ? "true" : "false"}
+    if (showInvestigationModalSelect) {
+      return <InvestigateDialogComponent/>;
+    } else if (showInvestigationPanel) {
+      return <InvestigationPanelComponent/>;
+    } else {
+      return <div/>;
```

Comment:
I'm not sure you need to return an empty div here

Reply:
We don't.  I originally had it showing text in the div for testing, but now it's just adding an unnecessary div.  Will remove.